src/libs/Clipboard/index.website.js

```diff
@@ -0,0 +1,80 @@
+import CONST from '../../CONST';
+import * as Browser from '../Browser';
+import MiscClipboardFunctions from './miscClipboardFunctions';
+
+const canSetHtml = MiscClipboardFunctions.canSetHtml;
+const setString = MiscClipboardFunctions.setString;
+
+/**
+ * Deprecated method to write the content as HTML to clipboard.
+ * @param {String} html HTML representation
+ * @param {String} text Plain text representation
+ */
+const copyToClipboardDeprecated = (html, text) => {
```

Comment:
```suggestion
const setHTMLSync = (html, text) => {
```

Please convert this arrow function to normal function.